modules/linear_boltzmann_solvers/a_lbs_solver/lbs_structs.h

```diff
@@ -95,10 +94,102 @@ enum SourceFlags : int
   ZERO_INCOMING_DELAYED_PSI = (1 << 6)
 };
 
-inline SourceFlags
-operator|(const SourceFlags f1, const SourceFlags f2)
+/**
+ * Source representation (a combination of `SourceFlag`s)
+ */
+struct Source
```

Comment:
Feel free to suggest a better name here. I used `Source` as in combination of `SourceFlag`s...